bears/general/URLBear.py

```diff
@@ -0,0 +1,155 @@
+import re
+import requests
+from urllib.parse import urlparse
+from aenum import Flag
+
+from coalib.bears.LocalBear import LocalBear
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.bearlib import deprecate_settings
+from coalib.results.HiddenResult import HiddenResult
+from coalib.settings.Setting import typed_list, typed_dict
+from coalib.parsing.Globbing import fnmatch
+
+
+class LINK_CONTEXT(Flag):
+    no_flags = 0
+    xml_namespace = 1
+    pip_vcs_url = 2
+    all_flags = 3
```

Comment:
https://bitbucket.org/stoneleaf/aenum says you can use bitwise comparison, which means this `all_flags` shouldnt be necessary.